src/app/games/rectanglz/RectanglzFailure.ts

```diff
@@ -0,0 +1,5 @@
+export class RectanglzFailure {
+
+    public static readonly MAX_DISTANCE_IS_: (n: number) => string = (n: number) => $localize`The maximum size of a jump is ${ n } step!`;
```

Comment:
ceci n'est pas au format "you did X, but should do Y". Ajoutez donc cela à la convention !

Reply:
C'est à systématiser à ce point ?
Genre, vous avez choisi un nombre trop grand, pour qu'il soit moins grand choisissez un nombre plus petit :D ?

Reply:
bah "Vous avez essayé de sauter trop loin ! La taille maximum d'un saut est de N" c'est très bien oui !